test/client/integration.test.ts

```diff
@@ -22,20 +25,20 @@ describe("Client Integration", function() {
         const provider = signer.provider as providers.Provider;
         const genesisEth1Block = await provider.getBlockNumber();
         await deployKeyless(signer, false);
-        const group = Group.new({ n: 32 });
-        const storagePacker = await storageManagerFactory(group);
-        const storageSyncer = await storageManagerFactory(group);
+        const storagePacker = await storageManagerFactory();
+        const storageSyncer = await storageManagerFactory();
 
         const parameters = PRODUCTION_PARAMS;
         parameters.USE_BURN_AUCTION = false;
         parameters.GENESIS_STATE_ROOT = storagePacker.state.root;
```

Comment:
The `GENESIS_STATE_ROOT` here depends on storagePacker's genesis states, so that we should setup  storagePacker before setting up `GENESIS_STATE_ROOT`.

We should move stuff in line 51~80 to before line 31

The error is not failing the test because #559